typescript/packages/subsurface-viewer/src/layers/grid3d/fragment.fs.glsl.ts

```diff
@@ -5,29 +5,33 @@ precision highp float;
 
 in vec3 cameraPosition;
 in vec4 position_commonspace;
-in float property;
+flat in float property;
+in float property_interpolated;
```

Comment:
Be aware that this means duplication of data sent to the fragment shader, with possible memory impact.

Let's finalize this PR as is, but we might want to create 2 files, for 2 different shaders: one flat version, one regular version...
You can resolve this comment